python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -158,6 +222,9 @@ def get_nodes_to_launch(
             self.max_workers,
             self.head_node_type,
             ensure_min_cluster_size,
+            utilization_scorer=partial(
+                self.utilization_scorer, node_availability_summary=None
```

Comment:
In what context will we be able to insert a non-None node_avaialability_summary?

Reply:
Also, it would be great if there were some way of arranging things to avoid the use of `partial`.
Better to be as unsophisticated as possible.

Reply:
oversight on my part, actually this should never be `None`. (The type signature is correct). 

lemme try to avoid introducing `partial`. It should be doable, but may arguable be more ugly due to more plumbing.

Reply:
If it's more ugly, don't do it.